samples/attestation/common/crypto.h

```diff
@@ -54,7 +47,7 @@ class Crypto
      * decrypt decrypts the given data using current enclave's private key.
      * Used to receive encrypted data from another enclave.
      */
-    bool decrypt(
```

Comment:
Just curious: why did this need to change?

Reply:
I capitalized it to make the naming consistent with other functions, since everything else is capitalized.